Observing-Interface-Setup/nublado-notebook-environment-update.rst

```diff
@@ -0,0 +1,262 @@
+
+.. _Update-Notebook-Environment-in-Nublado-What-is-the-Nublado-Platform:
+
+The Nublado Platform
+--------------------
+
+The nublado platform is the initial entry point for users to interact with the Rubin Observatory control system.
+It is highly flexible, allowing you to work on one or more components to accomplish both, high and low level operations.
+
+A jupyter-lab environment is provided for users to launch notebooks. By default, the platform provides a set of environments with released software versions and tools, including:
+
+- the `LSST stack <https://pipelines.lsst.io>`__, 
+- interfaces (*e.g.*, `xml <https://ts-xml.lsst.io/>`__, `IDL <https://github.com/lsst-ts/ts_idl>`__, `SAL <https://ts-sal.lsst.io/>`__, `Salobj <https://ts-salobj.lsst.io/>`__), and
+- high-level control algorithms (*e.g.*, `ATCS <https://github.com/lsst-ts/ts_atcs>`__, LATISS, MTCS, ComCam).
+
+.. _Update-Notebook-Environment-in-Nublado-Prerequisites:
+
+Prerequisites
+^^^^^^^^^^^^^
+
+.. This section should provide simple overview of prerequisites before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Do not include actions in this section. Any action by the user should be included at the beginning of the Procedure section below. For example: Do not include "Notify specified SLACK channel. Confirmation is not required." Instead, include this statement as the first step of the procedure, and include "Notification to specified SLACK channel." in the Prerequisites section.
+.. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
+
+- You must be logged into the nublado system.
+
+.. figure:: ./_static/nublado-interface.png
+    :name: Nublado Interface
+
+    Screenshot of nublado interface after successful login.
+
+.. _Update-Notebook-Environment-in-Nublado-Post-Conditions:
+
+Post-Condition
+^^^^^^^^^^^^^^
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+- The requested software version(s) is deployed for use in the jupyter-lab notebook. This will over-ride the versions available on the server.
+
+.. _Update-Notebook-Environment-in-Nublado-Procedure-Steps:
+
+Procedure Steps
```

Comment:
So this is an interesting use-case since there are a bunch of small procedures as one. I wonder if we should change the title/format a bit for this type of use-case?
I'm fine to just push this through for now, but thinking longer term i wonder if we could come up with something that's a little more clear.

Reply:
Yes, this is strange for two reasons: it is one rst file with multiple mini-procedures (even if they do similar things), and this is a difficult format for something that is extremely simple (3 instructions) from a usability standpoint. While it isn't necessarily the right approach, the steps should be written somewhere in a proper procedure, possibly using an example like it is written currently.

We will consider better formats/templates for this type of use-case while still providing the more rigid procedure format. For now, this version will be merged to facilitate discussion with content creators.